### PR TITLE
Add autoDocstring to recommended extensions

### LINKED ISSUE #13: Add autoDocstring to recommended extensions
The [autoDocstring](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) extension allows automatic generation of docstrings for methods, and supports Sphinx typeless style out of the box. We should add this to the recommended extensions, and add the recommended settings to the workspace config.

### PR DESCRIPTION
**Pull request by SaschaCowley**
Closes #13
Added [autoDocstring - Python Docstring Generator](https://marketplace.visualstudio.com/items?itemName=njpwerner.autodocstring) to recommended extensions, and set it up to use [Sphinx style](https://sphinx-rtd-tutorial.readthedocs.io/en/latest/docstrings.html) without types per the [coding standards](https://github.com/nvaccess/nvda/blob/master/projectDocs/dev/codingStandards.md).

### CODE REVIEW

File: extensions.json [hunk 1]
```diff
@@ -10,7 +10,8 @@
 		"davidanson.vscode-markdownlint",
 		"ms-vscode.powershell",
 		"charliermarsh.ruff",
-		"editorconfig.editorconfig"
+		"editorconfig.editorconfig",
+		"njpwerner.autodocstring",
```
Comment (seanbudd) [inline extensions.json:14]: I don't think terminating arrays with commas is valid JSON

https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Trailing_commas

```suggestion
		"njpwerner.autodocstring"
```

**Review by seanbudd — APPROVED** [review]

File: extensions.json (hunk 1 above)
Comment (SaschaCowley) [reply extensions.json:14]: This is true of standard JSON. These files are not standard JSON though (standard JSON doesn't support comments, either), and they do support terminating arrays with commas. See, for example, https://github.com/microsoft/vscode/issues/102061#issuecomment-656770244
> However, in our VSCode settings files, trailing commas are accepted.

Comment (SaschaCowley) [reply extensions.json:14]: Since this is allowable syntax and makes diffs cleaner, I think we should use it.

Comment (SaschaCowley) [reply extensions.json:14]: Unless you're concerned about external parsers choking on it? But if that's a concern we shouldn't allow comments either.

Comment (seanbudd) [reply extensions.json:14]: makes sense to me

**Pull request merged**